doc/adr/0006-project-naming.md

```diff
@@ -0,0 +1,31 @@
+# 6. Use product prefixes for project naming, if necessary
+
+Date: 2019-12-06
+
+## Status
+
+Proposed
+
+## [Context](https://docs.google.com/document/d/1ayJlohp2-s49c1_oypDE1bAfgYa5shR0I1GM6KRzDNA/edit)
+
+Libero projects may span a single product or possibly more, but they live in a single namespace of the [Libero Github organization](https://github.com/libero). Hence the projects need to:
+
+- be attributable to the originating product
+- avoid clashes in naming (e.g. many `article-store` projects)
+
+## Decisions
+
+- Name repositories that cover multiple products without prefixes (e.g. `infrastructure`, `community`)
+- Name internal and/or product-specific repositories with prefix for specific product (e.g. `reviewer-submission`, `my-product-storybook`)
+- Name generic repositories with no prefix (e.g. `event-bus`)
```

Comment:
What's the difference between "repositories that cover multiple products" (line 18) and "generic repositories"? Does it matter that these types are indistinguishable through this naming convention?

Reply:
I understood the former as repositories that are not built into the product (`infrastructure` and `community` as examples), whereas the latter as reusable blocks of the products that may be shared. Perhaps it's not an important distiction, or it needs better naming.

Reply:
`event-bus` seems a bad example.

Reply:
Agreed; do we have an existing example that better represents the category?

Reply:
Only archived things like https://github.com/libero/media-type I think.

Reply:
Will work with that